viewer/static/js/app/components/views/experiment-metadata.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export default function(props) {
+  let maybeName;
```

Comment:
Wonder if these should be initialized to empty strings. A good question for the React community really.